csp-tech-radar.csv

```diff
@@ -8,3 +8,40 @@ Long lived branches with Gitflow,hold,techniques,TRUE,"Gitflow is a strict branc
+Gatling,adopt,tools,FALSE,"An open-source load testing solution for load testing and integrates with development pipelines. <br /><br /><p><strong>Teams: </strong>Wrights, Olympus, Krakatoa, Everest, Rockies, Gonzales, Vestrahorn, K2, Rosa, Fuji</p>"
+Geb,trial,tools,FALSE,"A browser automation solution which can be used for scripting, scraping and general automation. <br /><br /><p><strong>Teams: </strong>Olympus, Rockies, Gonzales, Vestrahorn</p>"
+Hamcrest,trial,languages & frameworks,FALSE,"A framework that assists writing tests in several langs. It supports creating customized assertion matchers, allowing match rules to be defined declaratively.<br /><br /><p><strong>Teams: </strong>Vesuvius, K2, Rosa</p>"
+Jest,adopt,languages & frameworks,FALSE,A JavaScript testing framework designed to ensure correctness of any JavaScript codebase. <br /><br /><p><strong>Teams: </strong>Gonzales</p>
+Junit,hold,languages & frameworks,FALSE,"A unit testing open-source framework for the Java. This refers to JUnit older than version 4<br /><br /><p><strong>Teams: </strong>Olympus, Gonzales, Huascaran , Annapurna, Fuji, K2, Rosa</p>"
+Mockito,adopt,languages & frameworks,FALSE,"A Java-based framework used for unit testing.  The framework allows the creation of test double objects in automated unit tests for the purpose of test-driven development or behavior-driven development. <br /><br /><p><strong>Teams: </strong>K2, Rosa</p>"
+Moto,trial,tools,FALSE,"A python library for Mocking AWS Services that we will be using to simulate AWS resources.<br /><br /><p><strong>Teams: </strong>K2, Rosa</p>"
+Pact,assess,tools,FALSE,"A code-first tool for testing HTTP and message integrations using contract tests.<br /><br /><p><strong>Teams: </strong>Olympus, Krakatoa, Everest, Fuji</p>"
+PyTest,adopt,languages & frameworks,FALSE,"The Python testing framework. It can be used to write various types of tests, including unit tests, integration tests, end-to-end tests, and functional tests. <br /><br /><p><strong>Teams: </strong>K2, Rosa</p>"
+QTP,hold,tools,FALSE,A software that provides functional and regression test automation for software applications and environments. UFT supports keyword and scripting interfaces and features a graphical user interface. <br /><br /><p><strong>Teams: </strong>Olympus</p>
+Vitest,assess,languages & frameworks,FALSE,A JavaScript testing framework powered by Vite.<br />More teams should use<br /><br /><p><strong>Teams: </strong>Olympus</p>
+Wiremock,adopt,tools,FALSE,"A tool for building mock APIs<br /><br /><p><strong>Teams: </strong>Olympus, Krakatoa, Everest, Rockies, Gonzales, Vestrahorn, Vesuvius, K2, Rosa, Bianco</p>"
+Xunit,adopt,tools,FALSE,A unit testing tool for the .NET platforms.<br /><br /><p><strong>Teams: </strong>Olympus</p>
+AWS,adopt,platforms,FALSE,"Amazon Web Services<br /><br /><p><strong>Teams: </strong>Vesuvius, K2, Rosa, Bianco, Huascaran</p>"
```

Comment:
```suggestion
AWS,adopt,platforms,FALSE,"Amazon Web Services<br /><br /><p><strong>Teams: </strong>Vesuvius, K2, Rosa, Bianco, Huascarán</p>"
```

Reply:
The letter á causes encoding issues in csv files. Shall we still keep it?